operantanalysis/tests/test_operantanalysis.py

```diff
@@ -1,29 +1,40 @@
-from ..operantanalysis import accessfiles, rewardretrieval, respondingduringcueanditi, leverpressing
+from operantanalysis import load_file, extract_info_from_file, reward_retrieval, cue_iti_responding, lever_pressing, lever_press_latency
 
 
-def test_accessfiles():
-    (subjectnumber, timecode, eventcode) = accessfiles("../operantanalysis/sampledata/!2018-11-27_08h39m.Subject _0001.txt")
+def test_load_files():
+    (dictionary) = load_file("../operantanalysis/sampledata/!2018-11-27_08h39m.Subject _0001.txt")
+    assert "W" in dictionary
+    assert "Subject" in dictionary
+    assert len(dictionary) == 12
```

Comment:
I would really test your function more.  A bug in reading your custom data format would ripple through the entire analysis. I would have generated on the fly a few short but tricky cases, and verified that everything (including all data points) are read correctly

Reply:
I added another file for testing that has a trickier format to make sure that the loading still works, the data points are tested through a different function (extract data function), which would fail if the files are read incorrectly